testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -439,22 +439,23 @@ impl TxEmitter {
             info!("Not minting accounts");
             return Ok(()); // Early return to skip printing 'Minting ...' logs
         }
-        let num_seed_accounts = if requested_accounts / req.instances.len() > MAX_CHILD_VASP_NUM {
+        let mut num_seed_accounts = if requested_accounts / req.instances.len() > MAX_CHILD_VASP_NUM {
             requested_accounts / MAX_CHILD_VASP_NUM + 1
         } else {
             req.instances.len()
         };
         let num_accounts = requested_accounts - self.accounts.len(); // Only minting extra accounts
-        let num_new_child_accounts = (num_accounts + num_seed_accounts - 1) / num_seed_accounts;
         let coins_per_account = (SEND_AMOUNT + req.gas_price) * MAX_TXNS;
-        let coins_per_seed_account = coins_per_account * num_new_child_accounts as u64;
         let coins_total = coins_per_account * num_accounts as u64;
 
         let mut faucet_account = self.get_money_source(&req.instances, coins_total).await?;
         // Create seed accounts with which we can create actual accounts concurrently
         let seed_accounts = self
             .get_seed_accounts(&req.instances, num_seed_accounts)
             .await?;
+        num_seed_accounts = seed_accounts.len();
```

Comment:
without any comments this is hard to read.... it looks like the first value is `expected_num_seed_accounts` and the second value is `actual_num_seed_accounts` could you us either two variables or leavea comment on why we're resetting this value?

Reply:
oh, yea, I am going to add some comments

Reply:
because for premainnet, the seed account num is always 1.